### DISCUSSION TITLE
About the inferior mods

### DISCUSSION THREAD

**Discussion by Leroy8888** [HelixNebulaStudio/RiseOfTheDead#139] [General]
I realize I might've opened this post again, but as it currently stands we do have the inferior mechanic added into the game without any augments to adjust weapons for it. If it wasn't the intention to include augments in this patch maybe the inferior mechanic should also be excluded until augments are included along with it.

**Comment by MXKhronos** [comment]
As of right now, inferior mod mechanic has been reverted in dev branch back to inferior weapon. The idea was to even the playing field between high and low tier weapons to give lower tier weapons a bit more viability by reducing the impact of low tier mods on higher tier weapons. I know that was not received well.

It was a horizontal progression idea which in hindsight, didn't work well with the system currently exist where progression is built vertically and people move towards higher tier weapons inevitably. If the system was designed for more horizontal progression, weapon tier wouldn't be tied to power.

**Comment by Leroy8888** [comment]
I may be bold in my proposal, but if the horizontal progression idea is preference, why wouldn't you take the unique mechanics of the higher tiers and boost them in a way that they become more efficient at what they do for **some** aspects then inferior tiers. For example, reducing the damage of a SR while increasing the crit multiplier. Or increasing the focus potential of the rec-21 while lowering the reserve.

Mods could then be used to further amplify these effects. But compared to inferior tiers that would lack in some things. For example, a sr would not be as great for reliable DPS, while a rec-21 would not be able to sustain it's DPS for a prolonged time like the AWP would. The mods could also be used on inferior tiers and they would likely work on some, but to limited extent compared to that of what a higher tier _could_ achieve with it. Having mod empower weapon in a certain tier range is for that reason fine.

It's definitely not a simple task, absolutely not what I am saying and I will also require a lot of thinking. Especially since some stepping stones were set into place for powercreep to build up, to an eventual point where it seemed like this direction got changed. As it is also very likely this option was already considered long ago.
